x-pack/plugin/esql/qa/testFixtures/src/main/resources/lookup-join.csv-spec

```diff
@@ -395,78 +404,86 @@ emp_no:integer | language_code:integer | language_name:keyword
 ###########################################################################
 
 nullJoinKeyOnTheDataNode
-required_capability: join_lookup_v10
+required_capability: join_lookup_v11
 
 FROM employees
 | WHERE emp_no < 10004
 | EVAL language_code = emp_no % 10, language_code = CASE(language_code == 3, null, language_code)
 | LOOKUP JOIN languages_lookup_non_unique_key ON language_code
-| SORT emp_no
+| SORT emp_no, language_code, language_name
 | KEEP emp_no, language_code, language_name
 ;
 
 emp_no:integer | language_code:integer | language_name:keyword
-10001          | 1                     | [English, English, English]
-10002          | 2                     | [German, German, German]
+10001          | 1                     | English
+10001          | 1                     | English
+10001          | 1                     | English
+10001          | 1                     | null
+10002          | 2                     | German
+10002          | 2                     | German
+10002          | 2                     | German
 10003          | null                  | null
 ;
 
 mvJoinKeyOnTheDataNode
-required_capability: join_lookup_v10
+required_capability: join_lookup_v11
 
 FROM employees
 | WHERE 10003 < emp_no AND emp_no < 10008
 | EVAL language_code = emp_no % 10
 | LOOKUP JOIN languages_lookup_non_unique_key ON language_code
-| SORT emp_no
-| EVAL language_name = MV_SORT(language_name)
+| SORT emp_no, language_name
 | KEEP emp_no, language_code, language_name
 ;
 
 emp_no:integer | language_code:integer | language_name:keyword
 10004          | 4                     | Quenya
 10005          | 5                     | null
 10006          | 6                     | Mv-Lang
-10007          | 7                     | [Mv-Lang, Mv-Lang2]
+10007          | 7                     | Mv-Lang
+10007          | 7                     | Mv-Lang2
 ;
 
 mvJoinKeyFromRow
-required_capability: join_lookup_v10
+required_capability: join_lookup_v11
 
 ROW language_code = [4, 5, 6, 7]
 | LOOKUP JOIN languages_lookup_non_unique_key ON language_code
-| EVAL language_name = MV_SORT(language_name), country = MV_SORT(country)
 | KEEP language_code, language_name, country
```

Comment:
I just realized that this backport is still pending.
Do you mind adding `| SORT language_code, language_name, country` here to avoid flakiness with `multi-node`?

See https://github.com/elastic/elasticsearch/pull/120259

Reply:
Sure.

Reply:
Thank you!